fe/src/components/IssueTable/IssueTable.jsx

```diff
@@ -1,51 +1,108 @@
 import styled from 'styled-components';
+import { useContext } from 'react';
 import CheckBox from './CheckBox/CheckBox';
 import IssueItem from './IssueItem/IssueItem';
 import IssueStatusButtons from './TableToolBar/IssueStatusButtons/IssueStatusButtons';
 import TableFilterButtons from './TableToolBar/TableFilterButtons/TableFilterButtons';
 import TableToolBar from './TableToolBar/TableToolBar';
+import {
+  MainPageContext,
+  MainPageDispatchContext,
+  mainPageInitialState,
+} from '../../context/MainPage/MainPageContext';
+import { Button } from '../common';
+import { OPENED, RESET } from '../../constants';
+import { ReactComponent as xSquare } from '../../assets/xSquare.svg';
+import { setFilterOption } from '../../context/MainPage/MainPageActions';
 
-const IssueTable = ({
-  issues, counts, issueState, setFilterOptions,
-}) => {
-  const issueItems = issues?.length ? (
-    issues.map((issue) => <IssueItem key={issue.number} {...issue} />)
-  ) : (
-    <NoticeBox>검색과 일치하는 결과가 없습니다.</NoticeBox>
+const IssueTable = () => {
+  const { issues, filterOptions, loading } = useContext(MainPageContext);
+  const dispatch = useContext(MainPageDispatchContext);
+
+  let content;
+
+  if (loading) {
+    // TODO(덴): 스피너 기능에 대한 고민 필요. 굳이 필요한가?
+    // content = <Spinner />;
+  } else if (issues.length) {
+    content = issues.map((issue) => (
+      <IssueItem key={issue.number} {...issue} />
+    ));
+  } else {
+    content = <NoticeBox>검색과 일치하는 결과가 없습니다.</NoticeBox>;
+  }
+
+  const isFilterApplied = Object.values(filterOptions).some(
+    (option) => option !== null
+      && option !== false
+      && option.length > 0
+      && option !== OPENED,
   );
 
   return (
     <div>
+      {isFilterApplied ? (
+        <FilterNoticeBox>
+          <Button
+            type="ghostButton"
+            size="S"
+            gap="7px"
+            color="neutralText"
+            hoverColor="iconBackgoundBlue"
+            onclick={() => {
+              dispatch(
+                setFilterOption(RESET, mainPageInitialState.filterOptions),
+              );
+            }}
+          >
+            <XSquare />
+            현재의 검색 필터 및 정렬 지우기
+          </Button>
+        </FilterNoticeBox>
+      ) : (
+        ''
+      )}
       <TableToolBar>
         <CheckBox />
-        <IssueStatusButtons
-          {...counts}
-          issueState={issueState}
-          setFilterOptions={setFilterOptions}
-        />
+        <IssueStatusButtons />
         <TableFilterButtons />
       </TableToolBar>
-      <IssueItemList>{issueItems}</IssueItemList>
+      <IssueItemList>{content}</IssueItemList>
```

Comment:
이어지는 suggestion입니다.
```suggestion
    {issueItems ? (<IssueItemList>{issueItems}</IssueItemList>) : (<NoticeBox>검색과 일치하는 결과가 없습니다.</NoticeBox>)}
```
`IssueItemList`의 스타일링은 고려하지 않고 suggestion 드린 것이라 스타일링이 필요하면 추가될 수 있겠습니다!